modules/migration-configuring-cors-3.adoc

```diff
@@ -48,10 +48,10 @@ $ curl -v -k -X OPTIONS \
 "<cluster_url>/apis/migration.openshift.io/v1alpha1/namespaces/openshift-migration/migclusters" \ <1>
 -H "Access-Control-Request-Method: GET" \
 -H "Access-Control-Request-Headers: authorization" \
--H "Origin: https://<CAM_tool_url>" <2>
+-H "Origin: https://<CAM_web_console_url>" <2>
 ----
-<1> Update the source cluster URL.
-<2> Update the CAM tool URL.
+<1> Specify your CORS configured cluster's URL.
```

Comment:
Suggest rewording, this was a little hard to understand at first read through. Maybe "Specify the URL for your CORS-configured cluster."